Automattic/wp-calypso
# Multi-Domain: Added experiment to multi-domain flow

Related to 4293-gh-Automattic/dotcom-forge

## Proposed Changes

* Connected Multi-Domain flow to the experiment stead of config flag

## Testing Instructions



* Apply this diff: D126351-code
* Open this experiment on Abacus: `calypso_onboarding_domains_multi_domain_selection`
* Follow this guide (pb5gDS-2nD-p2#manually-assigning-a-user-to-an-experiment) to manually test the experiment
* Ensure you get the production interface if you have the `control` flag, and the Multi-Domain interface if you are at the `treatment` group

## Pre-merge Checklist
